tck/src/main/java/ee/jakarta/tck/concurrent/spec/Platform/dd/DeploymentDescriptorTestBean.java

```diff
@@ -56,36 +56,36 @@ public void testDeploymentDescriptorDefinesManagedScheduledExecutor() {
             };
 
             ManagedScheduledExecutorService executor = InitialContext
-                    .doLookup("java:global/concurrent/ScheduledExecutorD");
+                    .doLookup("java:app/concurrent/ScheduledExecutorE");
```

Comment:
It is confusing for a test that is named "testDeploymentDescriptorDefinesManagedScheduledExecutor" to be switching to test a ManagedScheduledExecutorService that is defined in an annotation rather than a deployment descriptor.   Also, by repurposing this test, are we losing covering of the deployment descriptor scenario?